tests/__init__.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from mock import Mock, patch
+
+from awscli.compat import six
+
+
+class MockStdIn(object):
```

Comment:
Testing helpers go in `awscli/testutils.py`.  Please move this to that file.  There's also a few existing helpers such as `capture_stdout()` that you might be able to refactor to share code.  They do a similar thing except with output.